discovery/gossiper.go

```diff
@@ -2357,13 +2357,61 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(
 				source: nMsg.source,
 				msg:    e1Ann,
 			})
+
```

Comment:
I don't think this needs to be done within the `if e1ann != nil`? Doesn't seem like a channel update us needed for the `node_anncounement` to be valid.